var/spack/repos/builtin/packages/cbtf-argonavis-gui/package.py

```diff
@@ -49,20 +49,55 @@ class CbtfArgonavisGui(QMakePackage):
     """
     homepage = "http://sourceforge.net/p/cbtf/wiki/Home/"
 
+    version('1.3.0.0', branch='1.3.0.0',
+            git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git')
+
     version('1.3.0', branch='master',
             git='https://github.com/OpenSpeedShop/cbtf-argonavis-gui.git')
 
-    depends_on("cmake@3.0.2:", type='build')
-    depends_on("openspeedshop+cuda gui='qt4'")
-    depends_on('qt@4.8.6:')
-    depends_on("boost@1.50.0:")
-    depends_on("cbtf")
-    depends_on("cbtf-krell")
-    depends_on("cbtf-argonavis")
+    version('develop', branch='master',
```

Comment:
@scheibelp We left those in because without that in we can't remove those old 1.3.0 packages.  Spack gets an abort when we try to remove the old packages.  So, we left those in even though they point to the same "master" version.

Reply:
Ah got it - we should make that easier to do

Reply:
@jgalarowicz I just tried uninstalling a package after removing the version and that appeared to work. Could you post the error message you get when you try this?